lib/objectbox-model.json

```diff
@@ -111,7 +111,7 @@
     },
     {
       "id": "5:6999419589241537986",
-      "lastPropertyId": "4:2940873808771910881",
+      "lastPropertyId": "5:1520307525834954271",
```

Comment:
Any reason why the ObjectBox model file has changed? Should this be reverted?

Reply:
some in-between changes. Doesn't hurt but I'll roll back.